addons/account/tests/test_account_analytic.py

```diff
@@ -428,3 +428,33 @@ def test_analytic_applicability_multiple_prefixes(self):
 
         # This invoice should not be blocked, as all lines have plans
         invoice.with_context({'validate_analytic': True}).action_post()
+
+    def test_tax_line_sync_with_analytic(self):
+        """
+        Test that the line syncs, especially the tax line, keep the analytic distribution when saving the move
+        """
+        account_with_tax = self.company_data['default_account_revenue'].copy({'tax_ids': [Command.set(self.company_data['default_tax_sale'].ids)]})
+        move = self.env['account.move'].create({
+            'move_type': 'entry',
+            'line_ids': [Command.create({'account_id': account_with_tax.id, 'debit': 100})]
+        })
+
+        # Check that the balancing line and the tax line have been created
+        self.assertEqual(len(move.line_ids), 3)
+
```

Comment:
```suggestion
```
Since we'll have to resolve the conflicts, we might remove this line?
It's implied in the `assertRecordValues()` below (pure nit picking, feel free to ignore 🤓)

Reply:
Done, thanks!